### PR TITLE
17557 chp36 content updates

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request updates the chapter 36 form with various content tweaks based on feedback garnered in staging review. 
Specifically, everything in [this list](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16690) has been addressed **with the exception** of points made about the chapter 36 wizard, which may be housed in their own pull request, and questions 1 & 3-5 under the "Questions" section. Those require further conversation, and aren't necessarily considered to be launch blockers. 

## Testing done
- local, all e2e tests pass

## Screenshots
<details><summary>Introduction Page and help component update </summary>

Screen Shot 2020-12-16 at 4 09 30 PM
Screen Shot 2020-12-16 at 4 09 39 PM
Screen Shot 2020-12-16 at 5 10 53 PM
</details>

<details><summary>Form page updates</summary>

Screen Shot 2020-12-16 at 4 09 58 PM
Screen Shot 2020-12-16 at 4 10 05 PM
Screen Shot 2020-12-16 at 4 18 15 PM
Screen Shot 2020-12-16 at 4 23 26 PM
</details>

<details><summary>Confirmation Page</summary>

Screen Shot 2020-12-16 at 5 02 01 PM
Screen Shot 2020-12-16 at 5 00 23 PM
</details>

## Acceptance criteria
- [x] all changes have been made
- [x] all tests pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vre/28-8832/components/GetFormHelp.jsx [hunk 1]
```diff
@@ -6,9 +6,10 @@ import Telephone, {
 
 const GetFormHelp = () => (
   <p className="help-talk">
-    Need help filling out the form or have questions about eligibility? Please
-    call VA Benefits and Services at{' '}
-    <Telephone contact={CONTACTS.VA_BENEFITS} />.<br />
+    If you have questions or need help filling out this form, please call our
+    MYVA411 main information line at <Telephone contact={CONTACTS.VA_311} /> and{' '}
+    select 0. We’re here 24/7.
```
Comment (Mottie) [inline src/applications/vre/28-8832/components/GetFormHelp.jsx:11]: Check out this Slack conversation - https://dsva.slack.com/archives/C5AGLBNRK/p1608065931112100?thread_ts=1608064849.111800&cid=C5AGLBNRK - related to the `24/7` part... not sure if it's really needed, but letting you know.

**Review by Mottie — APPROVED** [review]
LGTM

File: src/applications/vre/28-8832/components/GetFormHelp.jsx (hunk 1 above)
Comment (micahchiang) [reply src/applications/vre/28-8832/components/GetFormHelp.jsx]: Thank you! updated the PR.

**Review by cohnjesse — APPROVED** [review]
LGTM

File: src/applications/vre/28-8832/components/GetFormHelp.jsx [hunk 2]
```diff
@@ -7,11 +7,11 @@ import Telephone, {
 const GetFormHelp = () => (
   <p className="help-talk">
     If you have questions or need help filling out this form, please call our
-    MYVA411 main information line at <Telephone contact={CONTACTS.VA_311} /> and{' '}
-    select 0. We’re here 24/7.
-    <br />
+    <span aria-label="my VA 4 1 1.">MYVA411</span> main information line at{' '}
+    <Telephone contact={CONTACTS.VA_311} /> and select 0. We’re here{' '}
```
Comment (Mottie) [inline src/applications/vre/28-8832/components/GetFormHelp.jsx:11]: I don't think these are the same numbers... sorry for the confusion!

Comment (micahchiang) [reply src/applications/vre/28-8832/components/GetFormHelp.jsx:11]: 🤷  Not sure. @peggygannon provided us with the number 800-698-2411, and it seems to line up with the number here - https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/contacts.js#L81

Comment (Mottie) [reply src/applications/vre/28-8832/components/GetFormHelp.jsx:11]: Weird that the label includes 311, but the number is 411...

**Pull request merged**